lib/src/LameOptions.ts

```diff
@@ -571,6 +571,9 @@ class LameOptions {
                 else if (key == "genre") {
                     arg0 = `--tg`;
                 }
+                else if (key == "artwork") {
+                    arg0 = `--ti`;
```

Comment:
Where can I find the documentation for the parameter "--ti"? In the man page this parameter is not found.

Reply:
Yeah, it seems to be missing there. I found it via `lame --help id3`.